css/index.css

```diff
@@ -1 +1,348 @@
+aside,
+details,
+figcaption,
+figure,
+footer,
+header,
+hgroup,
+menu,
+nav,
+section {
+  display: block;
+}
+body {
+  line-height: 1;
+}
+ol,
+ul {
+  list-style: none;
+}
+blockquote,
+q {
+  quotes: none;
+}
+blockquote:before,
+blockquote:after,
+q:before,
+q:after {
+  content: "";
+  content: none;
+}
+table {
+  border-collapse: collapse;
+  border-spacing: 0;
+}
+button {
+  border: none;
+}
+* {
+  box-sizing: border-box;
+}
+html {
+  font-size: 62.5%;
+}
+html,
+body {
+  font-family: "Roboto", sans-serif;
+}
+h1,
+h2,
+h3,
+h4,
+h5 {
+  font-family: "Indie Flower", cursive;
+}
+h1 {
+  font-size: 4rem;
+}
+h2 {
+  font-size: 3.2rem;
+  padding-bottom: 10px;
+}
+h4 {
+  font-size: 2.5rem;
+  padding-bottom: 10px;
+}
+p {
+  line-height: 1.5;
+  font-size: 1.6rem;
+  padding-bottom: 10px;
+}
+img {
+  max-width: 100%;
+  height: auto;
+}
+.container {
+  max-width: 800px;
+  width: 100%;
+  margin: 0 auto;
+}
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  flex-direction: row;
+  width: 100%;
+  margin-top: 25px;
+  padding-bottom: 10px;
+  border-bottom: 2px dashed #c0c0c0;
+}
+@media (max-width: 500px) {
+  header {
+    flex-direction: column;
+    align-items: center;
+  }
+}
+header nav {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 40%;
+}
+@media (max-width: 500px) {
+  header nav {
+    width: 100%;
+    justify-content: center;
+    align-items: center;
+  }
+}
+header nav a {
+  display: flex;
+  width: 35%;
+  text-decoration: none;
+  font-size: 1.5rem;
+  color: #212529;
+  justify-content: center;
+}
+@media (max-width: 500px) {
+  header nav a {
+    align-items: center;
+  }
+}
+div.logo-txt {
+  display: flex;
+}
+.footer {
+  width: 100%;
+  border-top: 2px dashed silver;
+  background: #ffebcd;
+}
+.footer p {
+  text-align: center;
+  color: #212529;
+  font-size: 1.6rem;
+  padding: 20px;
+}
+.home h1 {
+  margin-bottom: 15px;
+}
+.home .top-content {
+  display: flex;
+  flex-direction: column;
+  margin: 0px 20px 20px 20px;
+  border-bottom: 2px dashed #c0c0c0;
+  padding-bottom: 10px;
+}
+.home .top-content img {
+  object-fit: contain;
+  padding-bottom: 25px;
+}
+.home .content-section {
+  margin: 30px;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+}
+.home .content-section .mobile-1 {
+  display: none;
+}
+@media (max-width: 500px) {
+  .home .content-section {
+    flex-direction: column;
+  }
+  .home .content-section .desktop-1 {
+    display: none;
+  }
+}
+.home .content-section .text-content {
+  width: 48%;
+  padding-right: 1%;
+}
+@media (max-width: 500px) {
+  .home .content-section .text-content {
+    width: 100%;
+    order: -1;
+  }
+}
+.home .content-section .img-content {
+  width: 48%;
+  padding-left: 1%;
+}
+@media (max-width: 500px) {
+  .home .content-section .img-content {
+    width: 100%;
+    object-fit: cover;
+  }
+}
+.home .content-section .img-content img {
```

Comment:
When selecting a raw tags like the `img` tag here for example I would personally use `>` to be as specific as possible to make sure I don't accidentally mess with other tags.

Reply:
ah gotcha, I had it nested in the less file so I thought it would be specific enough.

Reply:
That makes sense. I was reading it as it was `CSS` haha.